theano/tensor/nnet/nnet.py

```diff
@@ -1809,11 +1714,14 @@ def local_advanced_indexing_crossentropy_onehot_grad(node):
 
     # Dimension check before substitution
     if labels.ndim == 1 and x_var.ndim == 2:
-        return [crossentropy_softmax_1hot_with_bias_dx(out_grad, sm, labels)]
+        ret = crossentropy_softmax_1hot_with_bias_dx(out_grad, sm, labels)
+        # The stack trace of output_grad, sm and labels are not added
+        # but may need to be added at a future point
```

Comment:
Actually, we may want to add _a_ stacktrace _to_ out_grad, sm, and labels at a future point.